src/client/component/ProblemRunner.ts

```diff
@@ -149,7 +146,7 @@ export class ProblemRunner extends ComponentBase<any> {
             });
             return accumulator;
         };
-        const collectedValues: { [id: string]: any } = collectValues((this as any).shadowRoot.getElementById('template'), {});
+        const collectedValues: { [id: string]: any } = collectValues(this.querySelector('#template'), {});
```

Comment:
`getElementById` is not part of the `HTMLElement` API as it is for `Document` of `ShadowRoot`.
Hence I reverted to querySelector.